napalm/base/mock.py

```diff
@@ -28,6 +28,9 @@
 from pydoc import locate
 
 
+inspect_getargspec = getattr(inspect, "getfullargspec", inspect.getargspec)
```

Comment:
Can you just add a comment on why we need this? So we remember it in the future.